src/sentry/utils/committers.py

```diff
@@ -165,7 +167,7 @@ def get_previous_releases(project, start_version, limit=5):
 
 
 def get_event_file_committers(project, event, frame_limit=25):
-    group = Group.objects.get(id=event.group_id)
+    group = Group.objects.get_from_cache(id=event.group_id)
```

Comment:
It's actually weird that we fetch the group here like this... I'd have assumed we'd use `event.group` and the group would have already been fetched.

Reply:
Looking into this later as discussed on Slack.